abcd.cpp

```diff
@@ -0,0 +1,7 @@
+#include <stdio.h>
+int main()
```

Comment:
why are you not giving a new line after `#include<stdio.h>`
```
#include<stdio.h>

int main()
```

Reply:
yes, you are right i have update it.